ui/lib/apps/ClusterInfo/components/HostTable.js

```diff
@@ -0,0 +1,219 @@
+import { getValueFormat } from '@baurine/grafana-value-formats'
+import client from '@lib/client'
+import { Bar, CardTableV2 } from '@lib/components'
+import { useClientRequest } from '@lib/utils/useClientRequest'
+import { Tooltip, Typography } from 'antd'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+const { Text } = Typography
+
+function filterUniquePartitions(items) {
+  return items.filter(
+    (x, i, a) => a.findIndex((y) => y.partition.path === x.partition.path) === i
+  )
+}
+
+export default function HostTable() {
+  const { t } = useTranslation()
+
+  const { data: tableData, isLoading } = useClientRequest((cancelToken) =>
+    client.getInstance().hostAllGet({ cancelToken })
+  )
+
+  const columns = [
+    {
+      name: t('cluster_info.list.host_table.columns.ip'),
+      key: 'ip',
+      minWidth: 100,
+      maxWidth: 150,
+      isResizable: true,
+      isCollapsible: true,
+      onRender: ({ ip, unavailable }) => {
+        if (unavailable) {
+          return (
+            <Tooltip
+              title={t('cluster_info.list.host_table.instanceUnavailable')}
+            >
+              <Text type="warning">{ip}</Text>
+            </Tooltip>
+          )
+        }
+        return ip
+      },
+    },
+    {
+      name: t('cluster_info.list.host_table.columns.cpu'),
+      key: 'cpu_core',
+      minWidth: 60,
+      maxWidth: 60,
+      isResizable: true,
+      isCollapsible: true,
+      onRender: ({ cpu_core }) =>
+        cpu_core !== undefined ? `${cpu_core} vCPU` : '',
+    },
+    {
+      name: t('cluster_info.list.host_table.columns.cpu_usage'),
+      key: 'cpu_usage',
+      minWidth: 80,
+      maxWidth: 100,
+      isResizable: true,
+      isCollapsible: true,
+      onRender: ({ cpu_usage }) => {
+        if (cpu_usage === undefined) {
+          return
+        }
+        const { system, idle } = cpu_usage
+        const user = 1 - system - idle
+        const title = (
+          <>
+            <div>User: {getValueFormat('percentunit')(user)}</div>
+            <div>System: {getValueFormat('percentunit')(system)}</div>
+          </>
+        )
+        return (
+          <Tooltip title={title}>
+            <Bar
+              value={[user, system]}
+              colors={['#667cff', '#ff4d4f']}
```

Comment:
Avoid hard code colors. https://ant.design/docs/spec/colors#header
Also you can write `[null, '...']` so that the first segment uses default color